bin/run_neuralnet.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # EXAMPLE USAGE 
-# bin/run_neuralnet.sh -local true -maxIter 100 -conf ../src/test/resources/configuration/neuralnet -input dolphin-dnn/src/test/resources/data/neuralnet  -timeout 800000
+# bin/run_neuralnet.sh -local true -maxIter 100 -conf dolphin-dnn/src/test/resources/configuration/neuralnet -input dolphin-dnn/src/test/resources/data/neuralnet  -timeout 800000 -logPeriod 1000
```

Comment:
@beomyeol Oh, I didn't mean to change the configuration file... Should I revert it back to `local`?